src/components/Avatar.js

```diff
@@ -38,21 +57,21 @@ function Avatar() {
     playerClicked.play();
     if (numberOfPlayers === "p1s") {
       avatarData.forEach((item, index) => (avatarData[index].selected = ""));
-      temp.selected = "pt1";
+      temp.selected = Player1.student_name === undefined?"Player 1":Player1.student_name;
```

Comment:
What should be the default value? "pt1" or "Player 1"?

temp.selected = Player1.student_name || "pt1"

Reply:
Actually it was an image "pt1"
i replaced name with Player 1

Reply:
not resolved?